http.cc

```diff
@@ -107,14 +119,18 @@ static void handle_event(void *context, void *data, int len)
         int v_len = MIN(ev->header.value_len, MAX_HDR_LEN);
         fprintf(out, "%" PRIu64 " %" PRIu64 " RxHeader   %.*s %.*s\n", ev->conn_id, ev->req_id, n_len, ev->header.name, v_len,
                 ev->header.value);
+    } else if (ev->type == HTTP_EVENT_SEND_RESP) {
+        fprintf(out, "%" PRIu64 " %" PRIu64 " TxStatus   %" PRIu32 "\n", ev->conn_id, ev->req_id, ev->http_status);
     }
 }
 
 static std::vector<ebpf::USDT> init_usdt_probes(pid_t h2o_pid)
 {
-    std::vector<ebpf::USDT> vec;
-    vec.push_back(ebpf::USDT(h2o_pid, "h2o", "receive_request", "trace_receive_request"));
-    vec.push_back(ebpf::USDT(h2o_pid, "h2o", "receive_request_header", "trace_receive_request_header"));
+    const std::vector<ebpf::USDT> vec = {
```

Comment:
FYI You can use `std::initializer_list` constructor as below since C++11(remove `=`). (Note: This reduces one constructor call rather than original code)

```c++
    const std::vector<ebpf::USDT> vec {
        ebpf::USDT(h2o_pid, "h2o", "receive_request", "trace_receive_request"),
        ebpf::USDT(h2o_pid, "h2o", "receive_request_header", "trace_receive_request_header"),
        ebpf::USDT(h2o_pid, "h2o", "send_response", "trace_send_response"),
    };
```

Reply:
That's neat. Thanks for the tip @syohex -- I'll add another commit.